src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -144,16 +144,13 @@ public async Task TestTaskList()
             Assert.Equal(solution.Projects.Single().Documents.Single().Id, documentId);
             Assert.Equal(1, items.Length);
 
+            var pos = new LinePosition(2, 3);
             Assert.Equal(new TaskListItem(
                 documentId: documentId,
                 priority: 1,
                 message: "HACK: Test",
-                mappedFilePath: null,
-                originalFilePath: "test1.cs",
-                originalLine: 2,
-                mappedLine: 2,
-                originalColumn: 3,
-                mappedColumn: 3), items[0]);
+                span: new("test1.cs", pos, pos),
+                mappedSpan: new("test1.cs", pos, pos)), items[0]);
```

Comment:
using the same filepath name for mappedSpan is hte equivalent of passing `null` for mappedFilePath as this will not give an actual mapped path.